### PR TITLE
[Tizen] Add example app showing ml-service and rpk updates 

### PR DESCRIPTION
**Pull request by anyj0527**
- Add a simple app showing how to use API `ml_service_new` and `ml_service_request`.
- This app also demonstrate that how rpk updates can be used in AI app.

Check the [README.md](https://github.com/anyj0527/nnstreamer-example/blob/update-ml-svc-example/Tizen.native/ml-service-example/README.md)

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: Tizen.native/ml-service-example/sum_and_add_app/src/sum_and_add.c [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+  ml_tensors_info_destroy (ad->output_info);
+  ml_service_destroy (ad->service_handle);
+}
+
+/** @brief Initialize the ml_service path */
+int
+init_ml_service (appdata_s *ad)
+{
+  int status = -1;
+  // const gchar *key = "sum_and_add";
```
Comment (again4you) [inline Tizen.native/ml-service-example/sum_and_add_app/src/sum_and_add.c:215]: trivial: What about removing the unused code?

**Review by again4you — APPROVED** [review]
Except for trivial issues, this PR is good material for newbies. :+1:

File: Tizen.native/ml-service-example/sum_and_add_app/src/sum_and_add.c (hunk 1 above)
Comment (anyj0527) [reply Tizen.native/ml-service-example/sum_and_add_app/src/sum_and_add.c]: Thanks, I removed the code.

**Review by niley7464 — APPROVED** [review]
It has become easier to understand ml-service and rpk updates 😀
LGTM!

**Pull request merged**